source/MaterialXCore/Interface.cpp

```diff
@@ -492,6 +492,17 @@ vector<TokenPtr> InterfaceElement::getActiveTokens() const
     return activeTokens;
 }
 
+void InterfaceElement::addTokens(StringResolverPtr& resolver) const
+{
+    // Apply any interface tokens to the filename
+    for (auto token : getActiveTokens())
```

Comment:
Is the use of getActiveTokens vs getTokens intentional? getTokens was used in GeomInfo above, should that also be getActiveTokens?

Reply:
Yes. inheritance seems only to exist for Interface elements and no GeomInfo so GeomInfo only has getTokens().